dtfs-central-api/api-tests/v1/tfm/amendments/tfm-amendments-post.api-test.js

```diff
@@ -0,0 +1,93 @@
+const wipeDB = require('../../../wipeDB');
+const app = require('../../../../src/createApp');
+const api = require('../../../api')(app);
+const CONSTANTS = require('../../../../src/constants');
+const { MOCK_DEAL } = require('../../mocks/mock-data');
+const aDeal = require('../../deal-builder');
+
+describe('/v1/tfm/facilities/:id/amendment', () => {
+  let dealId;
+
+  const mockUser = {
+    _id: '123456789',
+    username: 'temp',
+    roles: [],
+    bank: {
+      id: '956',
+      name: 'Barclays Bank',
+    },
```

Comment:
Do we need `partyUrn` here?

Reply:
we don't need it - it's just the api-test